tests/installation/licensing/verify_license_translations.pm

```diff
@@ -45,14 +45,13 @@ sub run {
 
     my @available_translations = @{$license_agreement_info->{available_languages}};
     foreach my $translation (@{$test_data->{license}->{translations}}) {
-        unless (first { $_ eq $translation->{language} } @available_translations)
-        {
+        my $found_lang = first { /^\Q$translation->{language}\E/ } @available_translations;
```

Comment:
why this is needed?

Reply:
We'd check if "English (UK)" and  "English (US)" with brackets in the new language combox.